_health-care/_js/components/financial-assessment/DeductibleExpensesSection.jsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 
+import ErrorableTextInput from '../form-elements/ErrorableTextInput';
+import { isBlank, isValidMonetaryValue } from '../../utils/validations';
+
 class DeductibleExpensesSection extends React.Component {
+  constructor() {
+    super();
+    this.validateMonetaryFields = this.validateMonetaryFields.bind(this);
+  }
+
+  validateMonetaryFields(value, message) {
```

Comment:
Ideally, name predicate functions (anything returning true/false) so they start with "is" or "has"